style.md

```diff
@@ -69,3 +69,8 @@ Note that if you do this you will need to use the string version of the type,
 def my_func() -> 'module.that.causes.cycle.MyClass':
     pass
 ```
+
+#### Nomenclature
+
+For a list of terms we prefer for consistency see 
```

Comment:
nit: should be a comma after consistency.  Also, I would use stronger language, such as, "For the rules governing naming guidelines, see..."

Reply:
reworded